packages/core/src/QuickLInks/index.js

```diff
@@ -0,0 +1,99 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import clsx from "clsx";
+
+import { Link, Typography } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: "100%",
+    [theme.breakpoints.up("md")]: {
+      paddingBottom: "0.5rem",
+    },
+  },
+  list: {
+    listStyle: "none",
+    padding: 0,
+    paddingTop: "1rem",
+    "& > li": {
+      marginTop: "2.3125rem",
+    },
+    [theme.breakpoints.up("md")]: {
+      "& > li": {
+        marginTop: "1.5rem",
+      },
+    },
+  },
+  link: {
+    ...theme.typography.caption,
+    textDecoration: "none",
+  },
+  title: {
+    lineHeight: "inherit",
+  },
+}));
+
+function QuickLinks({ options, linkComponent, links, title, ...props }) {
+  const classes = useStyles(props);
+  const LinkComponent = linkComponent || Link;
+
+  return (
+    <div className={classes.root}>
+      <Typography variant={options.title.variant} className={classes.title}>
```

Comment:
should the title be part of the quick links?

Reply:
@KobbyMmo  I do think so because we need a parent `title` to classify the `links` below it.

Reply:
@kobyMmo Also we can exclude the title if we don't need it,  it does depend on the data we are using  👇 

<img width="580" alt="Screenshot 2021-01-13 at 12 44 18" src="https://user-images.githubusercontent.com/3693563/104435171-1f66f680-559d-11eb-8396-36ef044e16d8.png">

<img width="1400" alt="Screenshot 2021-01-13 at 12 44 00" src="https://user-images.githubusercontent.com/3693563/104435175-2130ba00-559d-11eb-9d59-87f0c3331d3d.png">

Reply:
yea, I'm still thinking, it seems it makes sense for a list to have a title, but should it be bundled with it?